app/_how-tos/add-rate-limiting-for-a-consumer-with-kong-gateway.md

```diff
@@ -61,25 +65,28 @@ entities:
         key_names:
           - apikey
 {% endentity_examples %}
-{% endcapture %}
-{{ plugin | indent: 3 }}
 
-1. Create a [consumer](/gateway/entities/consumer/) with a key.
+## 2. Create a consumer
+
+Consumers let you identify the client that's interacting with Kong Gateway. 
+With the Key Authentication plugin enabled globally, the consumer needs an API key to access any Kong Gateway services.
+
+Append the following snippet to the `kong.yaml` file to create a [consumer](/gateway/entities/consumer/) with an API key:
```

Comment:
the `entity_examples` block renders a copy if `append_to_existing_section: true` , and this looks a bit weird
<img width="734" alt="Screenshot 2024-08-29 at 18 10 27" src="https://github.com/user-attachments/assets/fe6d37cd-4659-49ea-98f5-5187324e48c2">

Reply:
I noticed that too - let me remove my line.